CHANGELOG.md

```diff
@@ -1,17 +1,19 @@
 # Changelog
 All notable changes to this project will be documented in this file.
 
-## v1.2.2 - 2014-08-12
-
+## [Unreleased][unreleased]
 ### Added
 - This CHANGELOG file to help users track progress of this gem. More information can be found at http://keepachangelog.com/
-- Better support for parameters in short form. Now, a letter parameter generates a command line with only 1 dash.
+- Fix rspec deprecation warnings
+- Remove unnecessary ```ensure``` and unnecessary ```in_data``` parameter
```

Comment:
The ensure can be removed from the changelog...it's not important to a user.
The in_data is also under the covers and not needed for changelog.  In fact, it's worse like this because it sounds like we stopped supporting the in_data parameter to build_command_line.